### DISCUSSION TITLE
Welcome to infirmary-integrated Discussions!

### DISCUSSION THREAD

**Discussion by tanjera** [tanjera/infirmary-integrated#182] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by luna-ray-dot** [comment]
Hello Tangera, please I saw your project on github it looks amazing and I have to give your flowers thanks for giving such a great project to the world for free but when I tried making some changes I noticed a folder 3rd parties.7-Zip was encrypted by a password and required to personally run.Please can I have the password thats if it doesn't cause you any trouble
